src/tools/segmentation/CorrectionTool.js

```diff
@@ -0,0 +1,312 @@
+import external from '../../externalModules.js';
+import BaseTool from '../base/BaseTool.js';
+// Drawing
+import { draw, drawJoinedLines, getNewContext } from '../../drawing';
+import toolColors from '../../stateManagement/toolColors.js';
+import { scissorsEraseInsideCursor } from '../cursors';
+import { modules } from '../../store';
+import { fillInside } from './utils';
+
+/**
+ * @public
+ * @class CorrectionTool
+ * @memberof Tools
+ * @classdesc Tool for slicing brush pixel data
+ * @extends Tools.Base.BaseTool
+ */
+export default class CorrectionTool extends BaseTool {
```

Comment:
A lot of this tool is identitcal to `SegmentationTool`, which is fine, but we should reuse code. The CorrectionTool can simply extend the `ScissorsTool` and override the one function it needs to (`_applySegmentationChanges`).